docs/guide/blog.md

```diff
@@ -272,6 +272,19 @@ message QueryPostsResponse {
 Once the types are defined in proto files, we can implement post querying logic. In `grpc_query_posts.go`:
 
 ```go
+package keeper
+
+import (
```

Comment:
There are a lot of additions to the imports in this part. Since `Posts` is the only function in this file, we might add the whole file, so for everyone who does not have auto-import-completion, this helps prevent errors.

Reply:
Ok! 👍